samples/highcharts/accessibility/expose-as-group-only/demo.js

```diff
@@ -0,0 +1,45 @@
+Highcharts.chart('container', {
+    chart: {
+        type: 'column'
+    },
+    title: {
+        text: 'Daily Steps Count'
+    },
+    xAxis: {
+        categories: ['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun'],
+        accessibility: {
+            description: 'Day of the week'
+        }
+    },
+    yAxis: {
+        title: {
+            text: 'Steps'
+        }
+    },
+    tooltip: {
+        valueSuffix: ' steps'
+    },
+    series: [{
+        name: 'Emma',
+        data: [9438, 10439, 11023, 13204, 10392, 9201, 12039],
+        color: '#3A3691',
+        accessibility: {
+            description: 'Emma walked the most in total during the week with ' +
+            '75739 steps.',
```

Comment:
If the total could be calculated instead of hard-coded using [templating](https://www.highcharts.com/docs/chart-concepts/templating) then this could be a nice flex for using templating.